src/ensembl/src/content/app/entity-viewer/shared/helpers/entity-helpers.ts

```diff
@@ -18,13 +18,57 @@ import {
   Slice,
   SliceWithLocationOnly
 } from 'src/content/app/entity-viewer/types/slice';
-import { Transcript } from 'src/content/app/entity-viewer/types/transcript';
-import { Gene } from 'src/content/app/entity-viewer/types/gene';
+import { PhasedExon } from 'src/content/app/entity-viewer/types/exon';
 import {
   Product,
   ProductType
 } from 'src/content/app/entity-viewer/types/product';
 
+export type IsProteinCodingTranscriptParam = {
```

Comment:
Would it be better to define these types immediately before the respective functions so that it is more clear why it is named in such a way?

Reply:
Don't really have a strong opinion. Moved the types around if that helps. They still look ugly as sin 😄